inception-pdf-editor/src/main/js/pdfanno/src/core/src/UI/span.js

```diff
@@ -274,35 +274,35 @@ window.addEventListener('DOMContentLoaded', () => {
       if (spanAnnotation) {
         spanAnnotation.deselect()
       }
-    }
```

Comment:
@reckart as I only moved this code into an if clause, is it ok to not annotate this change with `//BEGIN INCEpTION` etc.?